javascript/.vscode/launch.json

```diff
@@ -1,41 +0,0 @@
-{
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
```

Comment:
Why don't we keep this file in the root path?

Reply:
Didn't notice that in the root path, this was in the `.gitignore`